inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -536,7 +536,12 @@ QueryNetworkResult clDNNEngine::QueryNetwork(const CNNNetwork& network,
     CLDNNPlugin::Config conf = _impl->m_config;
     UpdateConfig(conf, network, config);
 
-    Program prog;
+    if (m_defaultContext == nullptr) {
+        m_defaultContext.reset(new CLDNNRemoteCLContext(
```

Comment:
m.b. default context should be created during plugin creation?

Reply:
1) Default context is used unconditionally here, so no need to check
2) Fixed